gandalf.js

```diff
@@ -2,7 +2,7 @@
  * Extras Plugin
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.